src/palak/FormAssignment1.java

```diff
@@ -0,0 +1,114 @@
+		else
+			System.out.println("Test Fail Title is not " + title);
+
+		// Fill all the form
+		driver.findElement(By.id("first name")).sendKeys("Palash");
+		driver.findElement(By.id("last name")).sendKeys("Soni");
+		driver.findElement(By.id("E-mail")).sendKeys("Palash444@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Vodafone");
+		
+		//Selecting Gender Value
+		WebElement genderValue = driver.findElement(By.name("gender"));
+		genderValue.click();
+		if (genderValue.isSelected())
+			System.out.println("Test Pass \n2.Selected Gender is -> "
+					+ driver.findElement(By.name("gender")).getAttribute("value"));
+		else
+			System.out.println("Test Fail");
+		
+		//Selecting Region Value
+		Select regionValue = new Select(driver.findElement(By.name("continents")));
+		regionValue.selectByVisibleText("Europe");
+
+		//Selecting Experience Value
+		WebElement experienceValue = driver.findElement(By.id("entry"));
+		experienceValue.click();
+		if (experienceValue.isSelected())
+			System.out.println("Test Pass \n3.Total Years of Exp. is selected");
+		else
+			System.out.println("Test Fail");
+		
+		//Entering Know Language Value
+		driver.findElement(By.id("knownlanguage")).sendKeys("Java");
+		String langValue = driver.findElement(By.id("knownlanguage")).getAttribute("value");
+		if (!langValue.isEmpty())
+			System.out.println("Test Pass \n4.Known Language is -> " + langValue);
+		else
+			System.out.println("Test Fail Language is not entered");
+		
+		//Selection Learning Language
+		WebElement learningLangValue = driver.findElement(By.id("java"));
+		learningLangValue.click();
+		if (learningLangValue.isSelected() && learningLangValue.getAttribute("value").equals("python")) {
+			System.out.println("Test Pass \n5.Learning Language is -> " + learningLangValue.getAttribute("value"));
+		} else
+			System.out.println(
+					"5.Test Fail \nyou have selected wrong language -> " + learningLangValue.getAttribute("value"));
+		
+		//Selecting Dream company value
+		WebElement dreamCompanyValue = driver.findElement(By.id("google"));
+		dreamCompanyValue.click();
+		if (dreamCompanyValue.isSelected() && dreamCompanyValue.getAttribute("value").equals("Google"))
+			System.out.println("Test Pass\n6.Dream Company is -> " + dreamCompanyValue.getAttribute("value"));
+		else
+			System.out.println(
+					"6.Test Fail you have selected wrong company ->" + dreamCompanyValue.getAttribute("value"));
+		
+		//Clicking on Confirm details Check box
+		WebElement confirmDetailValue = driver.findElement(By.id("confirmDetails"));
+		confirmDetailValue.click();
+		if (confirmDetailValue.isSelected()) {
+			System.out.println("Test Pass\n7.Confirm Details is selected");
+		} else
+			System.out.println("Test Fail\n7. Confirm details is not selected");
+
+		// Reset Functionality
+		WebElement reset = driver.findElement(By.id("resetBtn"));
+		reset.click();
+		if (driver.findElement(By.id("Company Name")).getAttribute("value").isEmpty())
```

Comment:
Where is the validation for radio button and checkbox ?

Reply:
Added 
if (driver.findElement(By.id("Company Name")).getAttribute("value").isEmpty() && (!dreamCompanyValue.isSelected())
					&&(!learningLangValue.isSelected()))